src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -99,10 +99,18 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit, OnChanges
     }
 
     /**
-     * Starting an exercise is not possible in the exam, otherwise see exercise.utils -> isStartExerciseAvailable
+     * Starting an exercise is not possible in the exam or if it's a team exercise and the student is not yet assigned a team, otherwise see exercise.utils -> isStartExerciseAvailable
      */
     isStartExerciseAvailable(): boolean {
-        return !this.examMode && isStartExerciseAvailable(this.exercise, this.gradedParticipation);
+        const individualExerciseOrTeamAssigned = (!this.exercise.teamMode || this.exercise.studentAssignedTeamId) as boolean;
```

Comment:
I haven't seen `as boolean` in our code base yet. Usually I'd expect one of these two:
```suggestion
        const individualExerciseOrTeamAssigned = !!(!this.exercise.teamMode || this.exercise.studentAssignedTeamId);
```
```suggestion
        const individualExerciseOrTeamAssigned = (!this.exercise.teamMode || this.exercise.studentAssignedTeamId) ?? false;
```

Reply:
I get your point but duplicating all the logic in four places isn't ideal either and let to the bug we fixed recently.